hpc_benchmark/hpc_benchmark.py

```diff
@@ -458,6 +460,19 @@ def run_simulation():
     final_kernel_status = nest.kernel_status
     final_kernel_status["time_simulate"] -= time_simulate_presim
```

Comment:
```suggestion
```
We can remove this line now and add the "time_simulate" to the timers to subtract.

Reply:
see above